website/docs/r/cluster.html.markdown

```diff
@@ -167,6 +167,39 @@ resource "mongodbatlas_cluster" "cluster-test" {
 }
 ```
 
+### Example Advanced Configuration Options
+
+```hcl
+resource "mongodbatlas_cluster" "test" {
+  project_id   = "<YOUR-PROJECT-ID>"
+  name         = "cluster-advanced-configuration"
+  disk_size_gb = 10
+
+  replication_factor           = 3
+  backup_enabled               = false
+  auto_scaling_disk_gb_enabled = true
+  mongo_db_major_version       = "4.0"
+
+  // Provider Settings "block"
+  provider_name               = "AWS"
+  provider_disk_iops          = 100
+  provider_encrypt_ebs_volume = false
+  provider_instance_size_name = "M10"
+  provider_region_name        = "EU_CENTRAL_1"
+
+  // Advanced Configuration Options
```

Comment:
I'd prefer we document the options but not include them in a sample.  People are often used to just copying examples and these should only be used by those in the know.

Reply:
Oh, Thank you, It's a good point, okay I will remove the example, but I will try to test for M0/M2/M5 to ensure me because the documentation doesn't explain this 👍